CMakeLists.txt

```
@@ -575,11 +579,12 @@ if (PKG_CONFIG_FOUND)
   set(MLPACK_VERSION_STRING "@MLPACK_VERSION_STRING@")
   configure_file(
     ${CMAKE_CURRENT_SOURCE_DIR}/CMake/mlpack.pc.in
```

```suggestion
    ${MLPACK_SOURCE_DIR}/CMake/mlpack.pc.in
```
(small change, might as well change this too for consistency)